src/modules/auth/locales/en/translation.ts

```diff
@@ -1,8 +1,17 @@
 const enResource = {
-  title: 'Home.page.tsx',
+  title: 'Signup.page.tsx',
   usernameLabel: 'Trainerr',
   langButtonEng: 'English',
   langButtonSrb: 'Serbian',
+  firstNameLabel: 'First Name',
+  lastNameLabel: 'Last Name',
+  birthdayLabel: 'Birthday',
+  emailLabel: 'Email Address',
+  passwordLabel: 'Password',
+  placeholderFirstName: 'John',
```

Comment:
If you were naming translations like this `firstNameLabel` (`nameElement`), you should keep doing that... 
So this would be `firstNamePlaceholder`.
Or the other way around... Not sure what is better... And if someone with more experience with localizations with i18next suggests something. Maybe you should look up some tutorials and see.

P.S. Either way, having this small convention will lead to more readable code.